activiti-services/activiti-services-core/src/main/java/org/activiti/services/core/commands/StartProcessInstanceCmdExecutor.java

```diff
@@ -26,8 +28,10 @@ public Class getHandledType() {
     }
 
     @Override
-    public void execute(Command cmd) {
-        ProcessInstance processInstance = processEngine.startProcess((StartProcessInstanceCmd) cmd);
-        System.out.println(processInstance.getId() + " -  " + processInstance);
+    public void execute(StartProcessInstanceCmd cmd) {
+        ProcessInstance processInstance = processEngine.startProcess(cmd);
+        StartProcessInstanceResults cmdResult = new StartProcessInstanceResults(cmd.getId(),
```

Comment:
@erdemedeiros exactly.. In my opinion we should return this *Results objects in the Rest Endpoints as well. It doesn't make sense to return the whole process instance in my opinion. You can get all the ProcessInstance info if you know the ID.